lib/qbo_api/connection.rb

```diff
@@ -129,6 +136,12 @@ def entity_name(entity)
       singular(entity)
     end
 
+    def reset_connections
```

Comment:
this allows you to reset any cached values when modifying the middleware, got it

Reply:
Yes, and also for when the `refresh_token` or `access_token` change, since the `access_token` is added to the connection via the `:oauth2` middleware.